File: pit-git/src/main/java/com/example/MyRequest.java
package com.example;

import java.io.Serializable;
import com.google.common.base.Preconditions;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;


public class MyRequest implements Serializable {

   private static final long serialVersionUID = -3548858114709541512L;
   private Long userId;

   public void validate() throws IllegalStateException {
      Preconditions.checkState(userId != null);
   }

   /**
    * Getter for property 'userId'.
    *
    * @return Value for property 'userId'.
    */
   public Long getUserId() {
      return userId;
   }

   /**
    * Setter for property 'userId'.
    *
    * @param userId Value to set for property 'userId'.
    */
   public void setUserId(Long userId) {
      this.userId = userId;
   }

   class Sub{}
}

File: pit-kotlin/src/main/java/com/example/MyRequest.kt
package com.example

import java.io.Serializable

data class MyRequest(val userId:Long?) : Serializable {


    @Throws(IllegalStateException::class)
    fun validate() {

    }

    internal inner class Sub

    companion object {

        private const val serialVersionUID = -3548858114709541512L
    }
}